Tests/test_file_apng.py

```diff
@@ -249,8 +249,8 @@ def test_apng_mode():
         assert im.mode == "P"
         im.seek(im.n_frames - 1)
         im = im.convert("RGBA")
-        assert im.getpixel((0, 0)) == (0, 255, 0, 255)
-        assert im.getpixel((64, 32)) == (0, 255, 0, 255)
+        assert im.getpixel((0, 0)) == (255, 0, 0, 0)
+        assert im.getpixel((64, 32)) == (255, 0, 0, 0)
```

Comment:
Using https://pypi.org/project/apng/ to split [Tests/images/apng/mode_palette_alpha.png](https://github.com/python-pillow/Pillow/blob/master/Tests/images/apng/mode_palette_alpha.png) into frames, this matches the transparent third frame